sunpy/net/_attrs.py

```diff
@@ -286,3 +286,13 @@ class ExtentType(SimpleAttr):
     """
     The type of Extent; for example, "FULLDISK", "SYNOPTIC", "LIMB", etc.
     """
+    
+class Info(SimpleAttr):
```

Comment:
Am I correct in my understanding that this isn't a thing the user can search on? If that's the case we don't need this as attrs are only for query input.

Reply:
Correct. Currently Info is not searchable in the VSO. But with a new Data Model that we are currently working  on, we may well make it searchable. I did find that if I did not have entries in both net/_attrs.py and net/vso/attrs.py for Info, I got errors when executing a TAP query.

Reply:
> we may well make it searchable

I think I would rather add this at the point you do make this searchable, as at the moment it might just confuse people.

> I got errors when executing a TAP query

I would be interested to see what they are, as I think I can search for the TAP backed things with released sunpy?

Reply:
After testing removing Info from both net/_attrs.py and net/attrs.py, I found that all tests were fine. So I was mistaken about the need for Info in those files. We can add it back once we make Info, and other fields searchable in the VSO in Data Model 2.0